src/globals/theme/_index.scss

```diff
@@ -65,13 +68,15 @@ $theme: dark;
   @return if($theme == dark, $dark, $light);
 }
 
-// Outputs the relevant classes for each supported theme.
-@each $theme, $tokens in $security--themes {
-  @include security--component($name: $theme) {
-    @include security--generate-css-custom-properties($theme: $tokens);
-  }
-}
-
 @include security--component($name: input__background__color--light) {
   @include input__background-color;
 }
+
+@if feature-flag-enabled($feature: $theme__name) {
```

Comment:
Similar condition as line 29?

Reply:
@SimonFinney I know we talked about this part yesterday but now I was curious if perhaps you could add a quick comment above this block, describing why it was necessary after those imports?